app/(docs)/object-mount/linux/getting-started/configuring-credentials/page.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: Configuring Credentials
 docId: heefureS2iubahpi
+redirects:
+  - /object-mount/?platform=linux&help=credentials
+  - /object-mount?platform=linux&help=credentials
+  - /object-mount/linux/credentials
```

Comment:
Can you explain in the github issue body why do we need this redirects?

If this is approved, it will be good to add those reason to the commit message too.

Reply:
Added a link to slack. I didn't get a confirmation